plugins/bundle/status.go

```diff
@@ -65,13 +70,20 @@ func (s *Status) SetError(err error) {
 
 	if astErr, ok := cause.(ast.Errors); ok {
 		s.Code = errCode
+		s.HTTPCode = ""
 		s.Message = types.MsgCompileModuleError
 		s.Errors = make([]error, len(astErr))
 		for i := range astErr {
 			s.Errors[i] = astErr[i]
 		}
+	} else if httpErr, ok := cause.(download.HTTPError); ok {
```

Comment:
[nit] if you don't mind, I think the extra case warrants making this a `switch err := cause.(type)`